### PR TITLE
Add an example to payload documentation

### PR DESCRIPTION
**Pull request by KobOp**
Fixes [#121](https://github.com/eclipse-esmf/esmf-semantic-aspect-meta-model/issues/121)

### CODE REVIEW

**Review by Yauhenikapl — APPROVED** [review]
Looks good!

**Review by ysrbo — APPROVED** [review]

File: documentation/modules/ROOT/pages/payloads.adoc [hunk 1]
```diff
@@ -56,6 +56,47 @@ applied. The other way round they can also be used to describe a validation algo
 * Operations defined in the Aspect Model are not subject to serialization.
 * Events defined in the Aspect Model are not subject to serialization.
 
+== Example
+Aspect Model
+[source, turtle]
```
Comment (atextor) [inline documentation/modules/ROOT/pages/payloads.adoc:61]: Please change this to enable variable evaluation in the code block:
```suggestion
[source,turtle,subs="attributes+,+quotes"]
```

File: documentation/modules/ROOT/pages/payloads.adoc [hunk 2]
```diff
@@ -56,6 +56,47 @@ applied. The other way round they can also be used to describe a validation algo
 * Operations defined in the Aspect Model are not subject to serialization.
 * Events defined in the Aspect Model are not subject to serialization.
 
+== Example
+Aspect Model
```
Comment (atextor) [inline documentation/modules/ROOT/pages/payloads.adoc:60]: Make this line a third level header?
```suggestion
=== Aspect Model
```

File: documentation/modules/ROOT/pages/payloads.adoc [hunk 3]
```diff
@@ -56,6 +56,47 @@ applied. The other way round they can also be used to describe a validation algo
...
+   samm:preferredName "speed"@en ;
+   samm:characteristic :Speed ;
+   samm:exampleValue "0.5"^^xsd:float .
+
+:Speed a samm-c:Measurement ;
+   samm:preferredName "speed"@en ;
+   samm:dataType xsd:float ;
+   samm-c:unit unit:kilometrePerHour .
+----
+Payload
```
Comment (atextor) [inline documentation/modules/ROOT/pages/payloads.adoc:89]: Add a blank line before, then make this a third level header
```suggestion
=== Payload
```

**Review by atextor — CHANGES_REQUESTED** [review]

File: documentation/modules/ROOT/pages/payloads.adoc [hunk 4]
```diff
@@ -56,6 +56,47 @@ applied. The other way round they can also be used to describe a validation algo
 * Operations defined in the Aspect Model are not subject to serialization.
 * Events defined in the Aspect Model are not subject to serialization.
 
+== Example
+=== Aspect Model
+[source,turtle,subs="attributes+,+quotes"]
+----
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
```
Comment (atextor) [inline documentation/modules/ROOT/pages/payloads.adoc:63]: Please use the prefix variables in the same way, they are referred to in [Using Namespaces](https://eclipse-esmf.github.io/samm-specification/2.1.0/modeling-guidelines.html#using-namespaces):

```asciidoc
[source,turtle,subs="attributes+,+quotes"]
----
@prefix : <urn:samm:{example-ns}.myapplication:1.0.0#> .
@prefix samm: <{samm}> .
@prefix samm-c: <{samm-c}> .
@prefix samm-e: <{samm-e}> .
@prefix unit: <{unit}> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
----
```

**Review by atextor — CHANGES_REQUESTED** [review]

**Review by atextor — APPROVED** [review]

**Pull request merged**